server/bin/unittests

```diff
@@ -422,8 +422,10 @@ function _run_test {
     export _testhtml_sat=$_testroot/var-www-html-satellite
     export _testopt=$_testroot/opt/pbench-server
     export _testopt_sat=$_testroot/opt/pbench-server-satellite
+    export _testbucket=$_testdir_local/s3-backup/testbucket
 
     _setup_state ${testname}
+    mkdir -p ${_testbucket}
```

Comment:
Why do we need to unconditionally create this test bucket directory?  Can we add this to the tar balls instead so that we don't need to change so many gold files gratuitously?

Reply:
Actually, the state/test-*.setup scripts can do this instead.  You can write one for all the backup tests, and symlink it to the others.